catkit/emulators/iris_ao_controller.py

```diff
@@ -114,7 +114,7 @@ def Popen(self,
         return self
 
     def write(self, buffer):
-        if self.disable_hardware:
+        if not self.disable_hardware:
```

Comment:
@ivalaginja Why are we changing this?

Reply:
Because it makes things work. I have no idea how and why, but in this form it actually takes the commands and communicates with the simulated hardware. There might be a better way to fix this?

Reply:
If it's not working then it's because ``disable_hardware=True`` when you want it to be ``False``. This piece of logic was correct as is. If ``disable_hardware == True`` we don't want to write/talk to the hardware so this func just returns.

Reply:
Fixed in https://github.com/spacetelescope/catkit/pull/141

Reply:
So should I revert this or leave as is?

Reply:
This needs to be reverted.